cmd/cp-url-syntax.go

```diff
@@ -42,7 +43,7 @@ func checkCopySyntax(ctx *cli.Context) {
 	for _, srcURL := range srcURLs {
 		_, _, err := url2Stat(srcURL)
 		if err != nil {
-			fatalIf(err.Trace(srcURL), fmt.Sprintf("Unable to stat '%s'.", srcURL))
+			console.Fatalf("Source %s doesn't exist.\n", srcURL)
```

Comment:
@vadmeste does the above phrase look better? If so, I can update the PR with @fwessels' suggestion.